### ISSUE TITLE
Ctrl + n does not take you to navigaition  menu once it is open

### ISSUE THREAD

**Issue by GeorgeKerscher** [edrlab/thorium-reader#1412]
The first time you use CTRL + N it takes you to the navigation bar and pressing enter expands. When you go down to the table of contents button, it expands properly. Once you are reading in the book and you want to go back to the navigation menu, ctrl +n seems to do nothing. Found this in 1.6 and in my continuous build later than 1.6. I am using NVDA 2020.4

**Comment by danielweck** [comment]
Hello George, I am not able to reproduce this behaviour on MacOS, with or without Voice Over. I will need to check the Windows + NVDA combination.

**Comment by danielweck** [comment]
Hello again George, I think you stated in this other discussion thread that the keyboard shortcut wasn't working because of a focus issue (opened dialog):

> Oh, the open dialogue is why the shortcut keys don't work. I reported that as an issue. Now I know to hit escape if a shortcut does not work and try it again.

https://github.com/edrlab/thorium-reader/issues/1438#issuecomment-826394669
